src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -35,19 +35,19 @@ public string Id
         }
         public System.Diagnostics.ActivityIdFormat IdFormat { get { throw null; } }
         public string OperationName { get { throw null; } }
-        public System.Diagnostics.Activity Parent { get { throw null; } }
-        public string ParentId { get { throw null; } }
+        public System.Diagnostics.Activity? Parent { get { throw null; } }
+        public string? ParentId { get { throw null; } }
         public System.Diagnostics.ActivitySpanId ParentSpanId { get { throw null; } }
         public bool Recorded { get { throw null; } }
-        public string RootId { get { throw null; } }
+        public string? RootId { get { throw null; } }
         public System.Diagnostics.ActivitySpanId SpanId { get { throw null; } }
         public System.DateTime StartTimeUtc { get { throw null; } }
-        public System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> Tags { get { throw null; } }
+        public System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string?, string?>> Tags { get { throw null; } }
         public System.Diagnostics.ActivityTraceId TraceId { get { throw null; } }
-        public string TraceStateString { get { throw null; } set { } }
-        public System.Diagnostics.Activity AddBaggage(string key, string value) { throw null; }
-        public System.Diagnostics.Activity AddTag(string key, string value) { throw null; }
-        public string GetBaggageItem(string key) { throw null; }
+        public string? TraceStateString { get { throw null; } set { } }
+        public System.Diagnostics.Activity AddBaggage(string? key, string? value) { throw null; }
+        public System.Diagnostics.Activity AddTag(string? key, string? value) { throw null; }
+        public string? GetBaggageItem(string? key) { throw null; }
```

Comment:
```suggestion
        public string? GetBaggageItem(string key) { throw null; }
```

Reply:
For me the implementation allows search for null key, can be nullable
```csharp
public string? GetBaggageItem(string? key)
{
    foreach (KeyValuePair<string, string?> keyValue in Baggage)
        if (key == keyValue.Key)
            return keyValue.Value;
    return null;
}
```

Reply:
If we're saying that keys added can't be null (AddBaggage now takes `string key` rather than `string? key`), then I think it's better for `GetBaggageItem` to not allow null either, even if the implementation happens to allow it.

Reply:
Will update, thanks